lib/northbound.c

```diff
@@ -570,8 +571,14 @@ static void nb_config_diff(const struct nb_config *config1,
 				LYD_DIFF_DEFAULTS, &diff);
 	assert(!err);
 
-	if (diff && DEBUG_MODE_CHECK(&nb_dbg_cbs_config, DEBUG_MODE_ALL))
-		nb_config_diff_dnode_log("iterating diff", diff);
+	if (diff && DEBUG_MODE_CHECK(&nb_dbg_cbs_config, DEBUG_MODE_ALL)) {
+		char *s;
```

Comment:
empty line between this and the first statement

Reply:
done